### PR TITLE
regulation: Add Red Packet number suggestions and format file

### PR DESCRIPTION
**Pull request by 0xPoe**
[rendered](https://github.com/hi-rustin/chengdu/blob/master/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,15 +10,17 @@
...
-  - 正二奎：明婷饭店
+    - 负四奎：奎元馆虾爆鳝面
+    - 零奎：大米先生
+    - 正一奎：兵哥碗杂面/关刀冒菜
+    - 正一点五奎：牛水煮/秋金小炒
+    - 正二奎：明婷饭店
 
 ## 新同事入群规定
 
 - 发红包，不得低于 200 微尹（实习生不用）
```
Comment (lichunzhu) [inline README.md]: duplicate
```suggestion
- 发红包，不得低于 200 微尹
```

File: README.md [hunk 2]
```diff
@@ -10,15 +10,17 @@
...
+    - 实习生不用
+    - 红包数量可以控制到比较刺激（推荐只发一个）
```
Comment (3AceShowHand) [inline README.md]: 要不 10 ～ 20 个吧。 1 个对手残不好啊。

**Review by lichunzhu — COMMENTED** [review]
Rest LGTM

**Comment by 3AceShowHand** [comment]
Rest LGTM

File: README.md (hunk 2 above)
Comment (0xPoe) [reply README.md]: Addressed. Thanks for your review!

**Review by 3AceShowHand — APPROVED** [review]

**Review by lonng — APPROVED** [review]

**Pull request merged**